docs/services/robot-service.md

```diff
@@ -3,11 +3,12 @@ title: "Robot Service"
 linkTitle: "Robot"
 description: "The Robot Service consists of a set of robot monitoring and management APIs that most robots support."
 type: "docs"
-draft: "true"
+draft: "false"
 weight: 10
 tags: ["robot state", "services"]
 # SME: Cheuk
 ---
+
 Robot Service constitutes a minimal set of APIs that most robots (Viam Server, [Viam Python SDK](https://python.viam.dev/) and various SDKs) should support.
 Users will likely use the Robot Service as an entrypoint to interact with Viam robots and provide a way to get updates from the robot as a whole.
```

Comment:
```suggestion
Use the Robot Service as an entrypoint to interact with Viam robots and provide a way to get updates from the robot as a whole.
```
Needs to speak to users instead of at them. I'm not opposed to putting the robot service page back up with the current state of our docs, but would emphasize that this whole part was clearly written for internal use so needs to be reworded if we want to have it client-facing.

Reply:
Yeah totally; sorry didn't mean to include this file in this PR